src/Simulation/Simulators.Tests/QuantumSimulatorTests/BasicTests.cs

```diff
@@ -55,7 +65,13 @@ where op.IsSubclassOf(typeof(AbstractCallable))
         [Fact]
         public void QSimX()
         {
-            using (var sim = new QuantumSimulator(throwOnReleasingQubitsNotInZeroState: false))
+            var simulators = new CommonNativeSimulator[] { 
```

Comment:
Similarly here; looks like the `using` issue affects most of the tests in this file, won't comment on further examples.